code/game/objects/structures/aliens.dm

```diff
@@ -154,29 +309,74 @@
 		return
 
 	for(var/turf/T in U.GetAtmosAdjacentTurfs())
-
 		if(locate(/obj/structure/alien/weeds) in T || isspaceturf(T))
 			continue
-
 		new /obj/structure/alien/weeds(T, linked_node)
 
 /obj/structure/alien/weeds/temperature_expose(datum/gas_mixture/air, exposed_temperature, exposed_volume)
 	..()
 	if(exposed_temperature > 300)
 		take_damage(5, BURN, 0, 0)
 
-//Weed nodes
+/*
+ * Wall Weeds
+ */
+
+/obj/structure/alien/wallweed
```

Comment:
What happens to wallweeds if the wall is destroyed?

Reply:
For now, nothing.
Namely because they still need a lot of work after this PR, and it'd be best to keep it to another PR later down the line, alongside with the fact that I can't particularly implement it with my current knowledge; What I could do is make a wall breaking break the wall weeds, this would break both weeds in single tile corridors, and also break wall weeds that are sharing a corner (and therefor the same spot) too. 
I'll resolve this, but feel free to re-open or poke me if there's a solution for this I'm unaware of that doesn't require a massive ammount of work, which will be handled later down the line 👍